### DISCUSSION TITLE
点击一次MenuItem 触发两次Click事件

### DISCUSSION THREAD

**Discussion by fopdoodle8** [BYJRK/DotNet-Discussions#97] [WPF 开发]
我在Menu里面的MenuItem 添加了Command 用于相应点击事件，但是点击一次MenuItem，它触发了我两次的Click事件，
我又给这个MenuItem绑定了isEnable属性，在进入点击事件的时候将enbale属性设置为false(之后没设置回true) 但是点击一次后它还是触发了两次点击事件，并且在触发两次事件之后MenuItem的Enable属性确实变为False了。
如何处理这个点击事件的问题，请教一下大佬们：）
`        bool canExecute() => Enable;

        [RelayCommand(CanExecute = nameof(canExecute))]
        private void DrawLine()
        {
            enable = false;
            Console.WriteLine("Draw Line");
        }
`

**Comment by BYJRK** [comment]
你XAML里面的绑定是怎么写的？

**Reply by fopdoodle8** [reply]
解决了：>
之前是用Click = “” 绑定，现在是用Command绑定就好了。
<MenuItem Header="Line" Command="{Binding DrawLineCommand}"/>
